Обновит файл README.md (#4)
htmlacademy/configs: readme.md

Собираем здесь конфиги для спеллеров и линтеров. Любое изменение через пулл реквест с как минимум одним апрувом от каждого направления.

## Установка:

> Перед установкой пакета необходимо настроить NPM, указав альтернативный репозиторий пакетов - GitHub Packages.

Reviewer: ```suggestion
Перед установкой пакета необходимо настроить NPM, указав альтернативный репозиторий пакетов — GitHub Packages.
```

### Получение Personal Access Token

> [Получение Personal Access Token (GitHub Docs)](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token)

Reviewer: ```suggestion
[Справка на GitHub Docs](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token)
```

> Для поиска и установки пакетов, при создании токена, необходимо выбрать область дейсвия токена:

Reviewer: ```suggestion
Для поиска и установки пакетов при создании токена необходимо выбрать область действия токена:
```

- [x] read:packages

> Если необходима возможность обновлять пакеты в дальнейшем, следует также выбрать область дейсвия токена:

Reviewer: ```suggestion
Если необходима возможность обновлять пакеты в дальнейшем, следует также довыбрать область действия токена:
```

- [x] write:packages

> ### Аутентификация с использование Personal Access Token

Reviewer: ```suggestion
### Аутентификация с использованием Personal Access Token
```

> [Аутентификация с использование Personal Access Token (GitHub Docs)](https://docs.github.com/en/packages/guides/configuring-npm-for-use-with-github-packages#authenticating-with-a-personal-access-token)

Reviewer: ```suggestion
[Справка на GitHub Docs](https://docs.github.com/en/packages/guides/configuring-npm-for-use-with-github-packages#authenticating-with-a-personal-access-token)
```

> * Первый вариант - с помощью конфигурационных файлов:

Reviewer: ```suggestion
* Первый вариант — с помощью конфигурационных файлов:
```

  * Создать файл `.npmrc` в домашней директории пользователя и добавить в него строку:

  ```bash
  //npm.pkg.github.com/:_authToken=TOKEN
  ```

  * Создать файл `.npmrc` в директории репозитория и добавить в него строку:

  ```bash

>   @htmlacademy:registry=https://npm.pkg.github.com/htmlacademy

Reviewer: Здесь `@htmlacademy` - это алиас спейса или имя пользователя, которое нужно подставить?

Author: Алиас спейса. Но я сейчас хочу попробовать обновить пакет без него. В документации указано что нужно добавить `registry=https://npm.pkg.github.com/OWNER`.

  ````

> * Второй варинат - с помощью инструментов командной строки NPM:

Reviewer: ```suggestion
* Второй вариант — с помощью инструментов командной строки npm:
```

>   * Воспользоваться командой `npm login` с флагом `scope`, который позволит расширить пространство поиска NPM пакетами указанного разработчика на GitHub Packages:

Reviewer: ```suggestion
  * Воспользоваться командой `npm login` с флагом `scope`, который добавит к результатам поиска пакеты указанного разработчика из GitHub Packages:
```

>   ``` bash

Reviewer: ```suggestion
  ```bash
```

  $ npm login --scope=@htmlacademy --registry=https://npm.pkg.github.com

  > Username: USERNAME
  > Password: TOKEN
  > Email: PUBLIC-EMAIL-ADDRESS
  ```

N.B. `TOKEN` заменить на полученный на предыдущем шаге

### Установка пакета

```bash

> npm install -DE @htmlacademy/configs  

Reviewer: ```suggestion
npm install -DE @htmlacademy/configs
```

```

> ## yaspeller

Reviewer: Нужно проверить, что такой переезд не изменил положение пакета в `node_modules`, или подправить инструкции по подключению конфига yaspeller
